### ISSUE TITLE
Img zoom not working inside modals

### ISSUE THREAD

**Issue by PixedBo** [Riron/ionic-img-viewer#132]
Hello, the plugin is working fine except inside modals. Is there a way to fix this?
Thank you!

**Comment by hitmacreed** [comment]
yes do this for eg: ` <ion-img  src="{{imgs}}"  imageViewer="{{imgs}}"></ion-img>`
